sdk/tests/client/addresses.rs

```diff
@@ -40,6 +40,21 @@ async fn addresses() {
     );
 }
 
+#[tokio::test]
+async fn evm_addresses() {
+    let secret_manager = crate::client::node_api::setup_secret_manager();
+
+    let opts = GetAddressesOptions::default()
+        .with_bech32_hrp(IOTA_TESTNET_BECH32_HRP)
+        .with_coin_type(IOTA_COIN_TYPE)
+        .with_range(0..1);
+    let public = secret_manager.generate_evm_addresses(opts.clone()).await.unwrap();
+    let internal = secret_manager.generate_evm_addresses(opts.internal()).await.unwrap();
+
+    assert_eq!(public[0], "0xb23e784f0464a30d536c961e414925eab6b3107d");
+    assert_eq!(internal[0], "0x98d8833ec4b82587d66207eb9c578fd0134c51b6");
```

Comment:
Where did you get these from?

Reply:
from generating them :-)

Reply:
Do you want me to add a comment like yours:
```
// Address generated with bip32 path: [??, 4218, 0, 0, 0].
// This address was generated with a MnemonicSecretManager.
// Seed: 0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2.
```

Reply:
Yeah but I mean, we should generate them with something else, otherwise the test is pointless

Reply:
not really, now its tested to be consistent in the future. What do you want me to generate it with? And whats the first number on BIP?

Reply:
Yeah but we shouldn't test it against itself. We should get the static values from another source to confirm it's correct.

Reply:
Just tried with https://iancoleman.io/bip39/ and it works with the seed in BIP39 Seed and `m/44'/4218'/0'/0` in BIP32 Derivation Path
![image](https://github.com/iotaledger/iota-sdk/assets/46689931/a8076a4a-904d-4a33-9ab4-b3756a982ba1)

Reply:
The comments I added in my PR are to explain I generated the values from a second source

Reply:
Thanks @Thoralf-M , then Brord maybe adapt the comment to say it's been tested against that

Reply:
Nice tool, added it to the list ;) will update